docs/doc_1_concepts.md

```diff
@@ -59,3 +59,47 @@ If you belive a method should be marked as stable, feel free to raise a PR appen
 ## Example
 
 To learn more about using the SDK, see the [basic example](https://github.com/mongodb/atlas-sdk-go/blob/main/examples/basic/basic.go).
+
+## Release Strategy (Semantic Versioning)
+
+Atlas Go SDK is following the Semantic Versioning (SemVer) specification. The versioning scheme used for this SDK is as follows: `vYYYYMMDDXXX.Y.Z`, where:
+
+- `YYYYMMDDXXX` represents the major version number and includes the date of the [Versioned API Resource Version](https://www.mongodb.com/docs/atlas/api/versioned-api-overview/) used in the SDK followed by three digits for other possible breaking changes.
+- `Y` represents the minor version number, indicating iterations of the same Versioned API Resource.
+- `Z` represents the patch version number, indicating non-user-affecting fixes in the SDK.
+
+### Versioning Rules
+
+#### Major Version (vYYYYMMDDXXX.0.0)
+
+A major version increment signifies significant changes and possible breaking changes in the SDK. The rules for major version increments are as follows:
+
+1. When the Versioned API version is introduced SDK undergoes breaking changes, the major version must be incremented.
```

Comment:
When an SDK release targets a new Versioned API resource version, the SDK's major version is mapped as follows: An API version `vYYYYMMDD` translates to SDK version `vYYYMMDD001.0.0`
For example, the API version `v20300101` translates to SDK version `v20300101001.0.0`